dracut-kdump.sh

```diff
@@ -312,11 +312,24 @@ do_final_action()
 
 do_dump()
 {
+	local _vmcore_creation_status
+
```

Comment:
`kdump.sh` runs in the initrd and needs to be POSIX compliant. You cannot use `local` here.

Reply:
Thanks, agreed.